core/lib/zksync_core/src/db_pruner/mod.rs

```diff
@@ -60,14 +86,18 @@ impl DbPruner {
             Arc::new(NextL1BatchWasExecutedCondition {
                 conn: connection_pool.clone(),
             }),
-            Arc::new(L1BatchOlderThanPruneCondition {
-                minimum_age: config.minimum_l1_batch_age,
-                conn: connection_pool.clone(),
-            }),
             Arc::new(ConsistencyCheckerProcessedBatch {
                 conn: connection_pool.clone(),
             }),
         ];
+        if config.minimum_l1_batch_age > Duration::ZERO {
```

Comment:
huh, now I understand why you wanted to set the default to zero there. Feel free to leave as is